internal/ngap/service/service.go

```diff
@@ -6,14 +6,14 @@ import (
 	"runtime/debug"
 	"time"
 
-	"git.cs.nctu.edu.tw/calee/sctp"
 	"github.com/sirupsen/logrus"
 
+	lib_ngap "github.com/free5gc/ngap"
```

Comment:
could we not use an alias?
which means just 
```go
"github.com/free5gc/ngap"
```

Reply:
line 14 also include tngf package named ngap. 

![image](https://github.com/free5gc/tngf/assets/160688822/093f52cb-3176-47f9-940c-6f80a03aa6ea)